### PR TITLE
refactor: Added Generic Input in place of variants[130]

### PR DESCRIPTION
**Pull request by Astitva877**
## Description
#130 
Please do not leave this blank 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/lib/components/inputs/Input.svelte [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  export let value: string = "";
```
Comment (LordNayan) [inline src/lib/components/inputs/Input.svelte:2]: Do not use explicit types @Astitva877 . 

For example, 
export let isRequired: boolean = false; (no need for boolean)
export let inputId: string = ""; (no need for string)

Comment (Astitva877) [reply src/lib/components/inputs/Input.svelte]: Resolved.

**Review by LordNayan — APPROVED** [review]

**Pull request merged**